ote/src/clj/ote/services/reports.sql

```diff
@@ -133,11 +135,13 @@ SELECT x.*,
                    AND ts.published IS NULL) AS "unpublished-services-count"
 
           FROM "transport-operator" op
-          JOIN "user" u ON u.name = (SELECT author
-                                       FROM "revision" r
-                                              JOIN "group" g ON op."ckan-group-id" = g.id
-                                      WHERE r.id = g."revision_id"
-                                      LIMIT 1)
+          JOIN "user" u ON u.name = (SELECT u.name
```

Comment:
Ok, but for clarity it would be good to not to use "`u`" twice as alias but for different tables.